ironfish/src/memPool/feeEstimator.test.ts

```diff
@@ -411,7 +405,7 @@ describe('FeeEstimator', () => {
         },
       ])
 
-      expect(fee).toBe(BigInt(6))
+      expect(fee).toBe(BigInt(9))
```

Comment:
@hughy or @ygao76 , could you double-check to make sure that this value seems reasonable? I'm not quite sure why it changed in this case, but I'd expect it to be pretty similar to the fees used in `transaction` and `transaction2`, so I think it's correct.